server/seed/demoSeed/generateLists.js

```diff
@@ -16,13 +16,18 @@ const generateLists = (demoUserId, userIds, cohortIds, items) => [
     favIds: [demoUserId],
     isArchived: false,
     isDeleted: false,
-    items: generateItems(demoUserId, userIds),
+    items: generateItems(demoUserId, userIds, [
+      '🍅 Tomato',
+      '🥓 Bacon',
+      '🧅 Onion',
```

Comment:
Onion emoji missing

Reply:
Isn't displayed? I  see Onion even on safari

Reply:
nope, No onion in the browser. 
Maybe replace with something diffrent